test/unit/gateways/global_collect_test.rb

```diff
@@ -35,6 +35,38 @@ def test_successful_authorize_and_capture
     assert_success capture
   end
 
+  def test_successful_authorize_and_capture_airline_fields
```

Comment:
Minor suggestion:
Consider renaming this test to `test_successful_purchase_with_airline_fields` since it uses the `purchase` call. (I realize that purchase is equivalent to authorize+capture on this gateway, but I found the naming to be initially confusing.)